setup.py

```diff
@@ -9,13 +9,13 @@
 
 setup(
     name='django-extra-fields',
-    version='1.2.4',
+    version='1.2.5',
     packages=['drf_extra_fields',
               'drf_extra_fields.runtests'],
     include_package_data=True,
     install_requires=['Django >= 1.4', 'djangorestframework >= 3.0.1'],
     extras_require={
-        "Base64ImageField": ["Pillow == 5.4.1"],
+        "Base64ImageField": ["Pillow == 6.2.0"],
```

Comment:
I think we can use `>=` for this. Anything between 6.2.0 and 7.0 is very likely to work without any issue.

Reply:
I think it's safer to have freezed dependencies not to be effected potential breaking changes.

Reply:
I agree with that but it's very unlikely to add a breaking change with minor version. Also, during this review, `6.2.1` is already released and it's a security update.

Reply:
Pillow version is updated to `6.2.1`, thank you!